keps/prod-readiness/sig-instrumentation/2831.yaml

```diff
@@ -3,4 +3,5 @@ alpha:
   approver: "@ehashman"
 beta:
   approver: "@wojtek-t"
-
+stable:
+  approver: "@wojtek-t"
```

Comment:
Can't comment on unchanged lines, so some comments heres:

1. "Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?"

Please update this section describing the manual tests that were run 
(https://github.com/mimowo/enhancements/blob/9303678d20aef13af6fdb85a98128deb7603cd94/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested is a nice example)


2. Are there any missing metrics that would be useful to have to improve observability

Any progress in having those metrics?

Reply:
I'll run through those steps, thanks.

Yes, there has been good progress on trace SDK metrics:

* Proposal for metric names: https://github.com/open-telemetry/semantic-conventions/pull/1631
* My prototype for Go: https://github.com/open-telemetry/opentelemetry-go/pull/6153

I expect those to be stabilized in the next ~3-6 months.

Reply:
I enabled, disabled, and re-enabled kubelet tracing in a kind cluster.  The cluster continued to function as normal.

Reply:
Great to hear about SDK metrics - can you please add this short info the the answer to the missing metrics question?

Reply:
updated to include that answer